core-plugins/docs/Excel-batchsource.md

```diff
@@ -32,6 +32,12 @@ in **filePath** input. (Macro-enabled)
 **memoryTableName:** KeyValue table name to keep the track of processed files. This can be
 a new table or existing one. (Macro-enabled)
 
+**memoryDatasetExplorer:** Name of the Explorer table corresponding to 'memoryTableName' dataset. If not mentioned,
```

Comment:
let's change this (and all other similar descriptions) to: "Name of the Hive table for the 'memoryTableName' dataset. By default, it is the same name as the dataset. (Macro-enabled)"